dbcon/execplan/calpontsystemcatalog.h

```diff
@@ -908,6 +927,15 @@ class CalpontSystemCatalog : public datatypes::SystemCatalog
   static uint32_t fModuleID;
 };
 
+// MCOL-5021
+const datatypes::SystemCatalog::ColDataType AUX_COL_DATATYPE = datatypes::SystemCatalog::UTINYINT;
+const int32_t AUX_COL_WIDTH = 1;
+// TODO MCOL-5021 compressionType is hardcoded to 2 (SNAPPY)
+const CalpontSystemCatalog::CompressionType AUX_COL_COMPRESSION_TYPE = CalpontSystemCatalog::COMPRESSION2;
```

Comment:
This compression type (which is set to 2) is for the AUX column in an actual user table, such as `test.t1`. While the `auxcolumnoid` is a column in the `calpontsys.systable` syscat table, where all columns are uncompressed.

Reply:
Ok, thanks, is it correct statement "that for each user column you about to create an additional aux column with associated EMEntry in extent map?"

Reply:
We are creating 1 AUX (hidden) column per user table.

Reply:
Ok, thanks, can you please share a link to the design doc (white paper) for this feature, I cannot find it in a Jira ticket, may be I'm missing something, but I only can see a "benchmarking file".

Reply:
I do not have a design doc for this feature yet, but I can write one.

Reply:
Yes, please, it would be nice to have one, because I need to update a "rebuildEM tool" based on your feature, but I have not been involved in design discussion and it's hard to understand a design based on just a code.